<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WINDOWS-BQKBTT7" name="Login tests" tests="3" failures="3" timestamp="1 Mar 2018 12:31:35 GMT" time="16.369" errors="0">
  <testcase name="login" time="6.854" classname="com.pack.home.TestExecution.TestRun">
    <failure type="java.lang.AssertionError" message="expected [
http://52.172.33.72:8080/ServicePortal/success.jsp] but found [http://52.172.33.72:8080/ServicePortal/sucess.jsp]">
      <![CDATA[java.lang.AssertionError: expected [
http://52.172.33.72:8080/ServicePortal/success.jsp] but found [http://52.172.33.72:8080/ServicePortal/sucess.jsp]
	at com.pack.home.TestExecution.TestRun.login(TestRun.java:49)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="trylogin" time="1.049" classname="com.pack.home.TestExecution.TestRun">
    <failure type="java.lang.AssertionError" message="Element not found">
      <![CDATA[java.lang.AssertionError: Element not found
	at com.pack.home.Libraries.LibraryForGenericFunction.waitAndClickForID(LibraryForGenericFunction.java:92)
	at com.pack.home.TestExecution.TestRun.trylogin(TestRun.java:37)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- trylogin -->
  <testcase name="logout" time="1.053" classname="com.pack.home.TestExecution.TestRun">
    <failure type="java.lang.AssertionError" message="Element not found">
      <![CDATA[java.lang.AssertionError: Element not found
	at com.pack.home.Libraries.LibraryForGenericFunction.waitAndClickForID(LibraryForGenericFunction.java:92)
	at com.pack.home.TestExecution.TestRun.logout(TestRun.java:55)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- logout -->
</testsuite> <!-- Login tests -->
